packages/ses/src/module-load.js

```diff
@@ -315,47 +382,80 @@ export const load = async (
     compartment,
   );
 
-  /** @type {Set<Promise<undefined>>} */
-  const pendingJobs = new Set();
   /** @type {Map<object, Map<string, Promise<Record<any, any>>>>} */
   const moduleLoads = new Map();
-  /** @type {Array<Error>} */
-  const errors = [];
 
-  const dependencyLoaded = memoizedLoadWithErrorAnnotation(
+  const { enqueueJob, drainQueue } = asyncOverseer();
+
+  enqueueJob(memoizedLoadWithErrorAnnotation, [
     compartmentPrivateFields,
     moduleAliases,
     compartment,
     moduleSpecifier,
-    pendingJobs,
+    enqueueJob,
+    (asyncImpl, _syncImpl) => asyncImpl,
     moduleLoads,
+  ]);
+
+  // Drain pending jobs queue and throw an aggregate error
+  const errors = await drainQueue();
+
+  throwAggregateError({
     errors,
+    errorPrefix: `Failed to load ${q(moduleSpecifier)} in compartment ${q(
+      compartmentName,
+    )}`,
+    debug: true,
```

Comment:
What's the recommended way to pass a boolean into here? Is there a precedent for using env variables? I keep switching between message and stack in these aggregated errors whenever working with SES.

Reply:
We do have a precedent for using `process.env` and some tooling around it from `@endo/env-options`. This is how we thread environment `LOCKDOWN_` options.

Reply:
`COMPARTMENT_LOAD_ERROR_VERBOSE` or `COMPARTMENT_LOAD_DEBUG` ?